airflow/www/templates/appbuilder/flash.html

```diff
@@ -53,13 +53,13 @@
       <div class="panel panel-default">
         <div class="panel-heading" id="errorHeading">
           <h4 class="panel-title">
-            <a id="alerts-accordion-toggle" data-toggle="collapse" data-parent="#accordion" href="#alerts" aria-expanded="false">
+            <a id="alerts-accordion-toggle" data-toggle="collapse" data-parent="#accordion" href="#alerts" aria-expanded="false" class="collapsed">
               DAG Import Errors ({{ dag_import_errors|length }})
```

Comment:
Since the red error message blocks won't be visible by default anymore, I think it would be good to add some emphasis to the to toggle with color and an icon—something like this:
![image](https://user-images.githubusercontent.com/3267/101212306-80b6b400-3646-11eb-8e66-0755c4e9b59f.png)
```suggestion
              <span class="text-danger"><span class="material-icons" aria-hidden="true">error</span> DAG Import Errors ({{ dag_import_errors|length }})</span>
```

Reply:
Yeah this looks a lot better, thanks for the suggestion!